backend/courses/util.py

```diff
@@ -257,7 +263,7 @@ def create_mock_data_with_reviews(code, semester, number_of_instructors):
         section.instructors.add(instr)
```

Comment:
What do these create mock xyz functions do/what are they used for?

Reply:
creating review data for courses tests to make sure reviews get aggregated properly -- not related to this PR

Reply:
That should probably get moved into a test file then, but it can probably just be done when moving the tests to a different folder